src/main/java/CLIOutput.java

```diff
@@ -37,43 +36,63 @@ public static void outputBoard (Board board, boolean symbol) {
         }
     }
     
-    private static getPieceSymbol (String pieceLetter, int[] pieceCoord) {
+    private static String getPieceSymbol (Piece piece) {
         // Returns the symbol for the piece
-        
-        //TODO: Figure out of piece if white or black!!!
 
-        switch(pieceLetter){
-            case "P":
-                return "♙";
-            case "R":
-                return "♖";
-            case "N":
-                return "♘";
-            case "B":
-                return "♗";
-            case "Q":
-                return "♕";
-            case "K":
-                return "♔";
-            case "p":
-                return "♟";
-            case "r":
-                return "♜";
-            case "n":          
-                return "♞";
-            case "b":
-                return "♝";
-            case "q":
-                return "♛";
-            case "k":
-                return "♚";
-            default:
-                if (pieceCoord[0]+pieceCoord[1] % 2 == 0) {
-                    return "■";
-                } else {
-                    return "□";
-                }
+        if (piece.getColor() == 'w') {
```

Comment:
Maybe we should consider creating constant values (possibly in the Player class or in the Piece class) to represent the possible colours considering they are only two? e.g. public static final char WHITE = 'w'